cmd/zc_enumerator.go

```diff
@@ -353,6 +353,10 @@ func InitResourceTraverser(resource common.ResourceString, location common.Locat
 		includeVersion = true
 	}
 
+	// print out version id when using azcopy list
+	if includeVersionsList {
```

Comment:
Is this feature behind a CLI flag or it will always print versions if available?

Reply:
this is technically a bug fix. it shouldnt go behind a flag

Reply:
It will only print out versions if specified by the properties flag by doing this: '--properties="VersionId"'